media_utils/video_puppeteer.py

```diff
@@ -148,6 +150,19 @@ def lag(self):
         elapsed_wall_time = clock() - self._start_wall_time
         return elapsed_wall_time - elapsed_current_time
 
+    def check_started(self):
+        '''
+        Note that Netflix videos don't start at zero by default, and
+        current_time does not get set until it does start.
```

Comment:
Confirming that I understand: `current_time` does not get set until the video starts?